content/docs/quickstart.mdx

```diff
@@ -22,74 +22,198 @@ keywords:
+:::note
 
-![Verify page](./img/quickstart/verify-service.png)
+If you haven't, install [**Pomerium CLI**](/docs/deploy/clients/pomerium-cli) with any of the following methods:
 
-Although identity verification failed, you successfully integrated Pomerium with the upstream verify service.
+<Tabs>
+<TabItem value="docker" label="Docker">
+
+```bash
+docker run cr.pomerium.com/pomerium/cli:latest
+```
 
-:::tip
+</TabItem>
+<TabItem value="yum" label="Yum">
 
-Because this guide doesn't include a [signing key](/docs/reference/signing-key) in the configuration, identity verification will fail.
+```bash
+[pomerium-pomerium]
+name=pomerium-pomerium
+baseurl=https://dl.cloudsmith.io/public/pomerium/pomerium/rpm/el/$releasever/$basearch
+repo_gpgcheck=1
+enabled=1
+gpgkey=https://dl.cloudsmith.io/public/pomerium/pomerium/gpg.6E388440B94E1407.key
+gpgcheck=1
+sslverify=1
+pkg_gpgcheck=1
+```
+
+</TabItem>
+<TabItem value="deb" label="Deb">
+
+```bash
+curl -1sLf 'https://dl.cloudsmith.io/public/pomerium/pomerium/gpg.6E388440B94E1407.key' | apt-key add -
+echo "deb https://dl.cloudsmith.io/public/pomerium/pomerium/deb/debian buster main" > /etc/apt/sources.list.d/pomerium-pomerium.list
+```
 
-See [Identity Verification](/docs/capabilities/getting-users-identity) for more information on how Pomerium can use JWTs for authentication.
+</TabItem>
+<TabItem value="binaries" label="Binaries">
+
+```bash
+ARCH=[your arch]
+OS=[your os]
+VERSION=[desired version]
+curl -L https://github.com/pomerium/cli/releases/download/${VERSION}/pomerium-cli-${OS}-${ARCH}.tar.gz \
+    | tar -z -x
+```
+
+</TabItem>
+<TabItem value="homebrew" label="Homebrew">
+
+```bash
+brew tap pomerium/tap
+brew install pomerium-cli
+```
+
+</TabItem>
+<TabItem value="source" label="Source">
+
+```bash
+git clone git@github.com:pomerium/cli.git
+cd pomerium
+make build
+./bin/pomerium-cli --help
+```
+
+</TabItem>
+
+</Tabs>
 
 :::
 
-## Next Steps
+### Connect to Verify
+
+In the Zero Console, select the **From** route for the Verify app.
 
-If you want to [try Enterprise](https://www.pomerium.com/enterprise-sales/), check out the [Enterprise with Docker quickstart](/docs/deploy/enterprise/quickstart).
+For example:
 
-If you want to try connecting Pomerium with other services, see some of our [Guides](/docs/guides).
+![Selecting the external URL for the Verify route in the Zero Console](./img/quickstart/pz-quickstart-verify-route.png)
 
-**Did you finish this quickstart guide?** We'd love to hear what you think. Get in touch with us on our [Discuss forum](https://discuss.pomerium.com/), message us on [Twitter](https://twitter.com/pomerium_io), [LinkedIn](https://www.linkedin.com/company/pomerium-inc), or check out our [Community](https://www.pomerium.com/docs/community) page.
+:::note
 
-:::caution
+Because the Verify service uses a publically accessible URL, Pomerium Zero won't be able to connect to the `jwks.json` endpoint to fetch the signing key used to verify the user's JWT.
```

Comment:
```suggestion
You will see the following error because the Verify service uses a publicly accessible URL. In the current setup, Pomerium Zero won't be able to connect to the `jwks.json` endpoint to fetch the signing key used to verify the user's JWT.
```